contents/blog/posthog-launchdarkly.md

```diff
@@ -0,0 +1,293 @@
+
+## How are PostHog Feature Flags different from LaunchDarkly Feature Flags?
+
+Feature Flags are, by and large, a commodity feature, meaning that the gap between one Feature Flag vendor and another, at least in terms of Feature Flags alone, is not going to be large. 
+
+Feature Flags in their most basic form, after all, are IF statements that you can flip to deploy code. The differentiation between Feature Flag vendors is what they offer and layer on top of this core functionality, including the UX and any additional, complementary features. 
+
+The differences between PostHog Feature Flags and LaunchDarkly Feature Flags, then, come less from the Feature Flags themselves than what and how those Feature Flags connect to other capabilities. 
+
+## PostHog vs. LaunchDarkly Feature Flag comparison
+
+On our [Feature Flags page](https://posthog.com/product/feature-flags), you can see a comparison table that compares PostHog to LaunchDarkly, Optimizely, Flagsmith, and Growthbook. The below focuses on the major similarities and differences between PostHog and LaunchDarkly in greater detail.
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td><strong>LaunchDarkly</strong>
+   </td>
+   <td><strong>PostHog</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Platform</strong>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Free plan
+   </td>
+   <td>✅
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Open source
+   </td>
+   <td>❌
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Self hostable
+   </td>
+   <td>❌
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Cloud hosting
+   </td>
+   <td>✅
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Product analytics
+   </td>
+   <td>❌
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Feature flags</strong>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Target by percentage
+   </td>
+   <td>✅
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Target by user properties
+   </td>
+   <td>✅
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Flag scheduling
+   </td>
+   <td>✅
+   </td>
+   <td>❌
+   </td>
+  </tr>
+  <tr>
+   <td>Experimentation
+   </td>
+   <td>✅
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Multivariate flags
+   </td>
+   <td>✅
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Unlimited flags for free
+   </td>
+   <td>❌
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>3rd party plugins for free
+   </td>
+   <td>✅
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Activity logs
+   </td>
+   <td>✅
+   </td>
+   <td>✅
+   </td>
+  </tr>
+  <tr>
+   <td>Data export
+   </td>
+   <td>✅
+   </td>
+   <td>❌
+   </td>
+  </tr>
+  <tr>
+   <td>Multi-environment support
+   </td>
+   <td>✅
+   </td>
+   <td>✅
```

Comment:
Since when does PostHog support multiple environments? Where can I find this feature?

Reply:
@neilkakkar may have the answer to this (which may be that we don't)

Reply:
I assume this refers to: https://docs.launchdarkly.com/sdk/features/multiple-environments

In which case, yep, we do, since you can arbitrary instantiate as many libraries with as many projects as you'd like :)

Reply:
My understanding that environments is a underlying feature of Projects: 

> Environments are organizational units contained within a project. To learn more, read [Environments](https://docs.launchdarkly.com/home/organize/environments).

Reply:
I might be mistaken, but given that environments all have their own keys, seems like they map directly to PostHog's projects? Or is there more to it?

Reply:
My understanding is that it are environments (e.g. staging, production) within a project. Instead of like in PostHog a project per environment. Launchdarkly allows you easily to change feature flags changes between environments while in PostHog you need to syncronise the feature flags manually between your environment projects